docs/kyma-environment-broker/03-01-service-description.md

```diff
@@ -121,14 +223,19 @@ These are the provisioning parameters for the Trial plan that you can configure:
 | ---------------|-------|-------------|----------|---------------|---------------|  
 | **name** | string | Specifies the name of the Kyma Runtime. | Yes | Any string| None |  
 | **region** | string | Defines the cluster region. | No | `europe`,`us`, `asia` | Calculated from the platform region |  
-| **provider** | string | Specifies the cloud provider used during provisioning. | No | `Azure`, `GCP` | `Azure` |
- 
+| **provider** | string | Specifies the cloud provider used during provisioning. | No | `Azure`, `AWS`, `GCP` | `Azure` |
+| **context.active** | string | Specifies if the SKR should be suspended or unsuspended. | `true`, `false` | None | 
+
 The **region** parameter is optional. If not specified, the region is calculated from platform region specified in this path:
 ```shell
 /oauth/{platform-region}/v2/service_instances/{instance_id}
 ```
-The mapping between the platform region and the provider region (Azure or GCP) is defined in the configuration file in the **APP_TRIAL_REGION_MAPPING_FILE_PATH** environment variable. If the platform region is not defined, the default value is `europe`.
+The mapping between the platform region and the provider region (Azure, AWS or GCP) is defined in the configuration file in the **APP_TRIAL_REGION_MAPPING_FILE_PATH** environment variable. If the platform region is not defined, the default value is `europe`.
 
  </details>
  </div>
 
+## Free plan
+
+Free plan allows you to install Kyma on Azure or AWS. The configuration and assumptions are the same as for the Trial plan.
```

Comment:
```suggestion
Free plan allows you to install Kyma on Azure or AWS. The configuration and assumptions are the same as for the trial plan.
```

Reply:
Are you sure about this sentence? AFAIK, the free plan is not uninstalled after 30 days. Let's confirm.